Sources/Commands/SwiftTool.swift

```diff
@@ -594,6 +594,33 @@ public class SwiftTool {
             throw error
         }
     }
+    
+    /// Evaluate extensions for any reachable targets in the graph, and return a mapping from targets to corresponding evaluation results.
+    func evaluateExtensions(graph: PackageGraph) throws -> [ResolvedTarget: [ExtensionEvaluationResult]] {
+        do {
+            // Configure the inputs to the extension evaluation.
+            // FIXME: These paths are still fairly preliminary.
+            let buildEnvironment = try buildParameters().buildEnvironment
```

Comment:
This part of the code will evolve in future PRs.  There are likely to be edge cases that need to be fixed before the feature can be enabled after approval.